new-components/enrichers/custom-annotation/README.md

```diff
@@ -0,0 +1,30 @@
+# custom-annotation
+
+This is component implements an [enricher](https://github.com/smithy-security/smithy/blob/main/sdk/component/component.go)
+that adds a custom json annotation to the fetched vulnerability findings associated with the workflow.
+
+## Environment variables
+
+The component used environment variables for configuration.
+
+It requires the component
+environment variables defined [here](https://github.com/smithy-security/smithy/blob/main/sdk/README.md#component) as well as the following:
+
+| Environment Variable       | Type   | Required | Default | Description                                 |
+|----------------------------|--------|----------|---------|---------------------------------------------|
+| CUSTOM\_ANNOTATION\_NAME     | string | yes      | -       | The name of the annotation to be added.     |
+| CUSTOM\_ANNOTATION\_VALUES   | string | yes      | {}      | Json annotations to be added as annotation. |
```

Comment:
It might be clearer to say they are in a key-value format and say e.g. `{foo:bar}`

Reply:
Done